### PR TITLE
Add pylint and config for pylint and travis

### PR DESCRIPTION
**Pull request by MasterChief-John-117**

### CODE REVIEW

File: proxstar/__init__.py [hunk 1]
```diff
@@ -194,7 +195,6 @@ def vm_details(vmid):
 @auth.oidc_auth
 def vm_power(vmid, action):
     user = User(session['userinfo']['preferred_username'])
-    proxmox = connect_proxmox()
```
Comment (devinmatte) [inline proxstar/__init__.py]: Um we probably want to run `connect_proxmox()`. The variable result `proxmox` is probably unnecessary

File: proxstar/tasks.py [hunk 2]
```diff
@@ -114,7 +112,6 @@ def process_expiring_vms_task():
     with app.app_context():
         proxmox = connect_proxmox()
         db = connect_db()
-        starrs = connect_starrs()
```
Comment (devinmatte) [inline proxstar/tasks.py]: Again we probably still want to connect to stars

File: proxstar/tasks.py [hunk 3]
```diff
@@ -158,7 +154,6 @@ def setup_template_task(template_id, name, user, ssh_key, cores, memory):
         db = connect_db()
         logging.info("[{}] Retrieving template info for template {}.".format(
             name, template_id))
-        template = get_template(db, template_id)
```
Comment (devinmatte) [inline proxstar/tasks.py]: This may be an indication of a bug. I feel like this should be used

File: proxstar/__init__.py (hunk 1 above)
Comment (MasterChief-John-117) [reply proxstar/__init__.py]: All three re-added without assignment

File: proxstar/__init__.py [hunk 4]
```diff
@@ -188,18 +191,19 @@ def vm_details(vmid):
             usage_check=usage_check)
     else:
         abort(403)
```
Comment (devinmatte) [inline proxstar/__init__.py]: I think you can return `abort(403)`, but that's minor

Comment (MasterChief-John-117) [reply proxstar/__init__.py]: Checked and that works. Change made in next commit

File: proxstar/__init__.py [hunk 5]
```diff
@@ -175,7 +178,7 @@ def hostname(name):
 @auth.oidc_auth
 def vm_details(vmid):
     user = User(session['userinfo']['preferred_username'])
-    proxmox = connect_proxmox()
+    connect_proxmox()
```
Comment (com6056) [inline proxstar/__init__.py]: Not sure we even need these calls, I think these were just left behind from when I moved a bunch of the logic into the user/vm classes. If we're not doing anything with the result of `connect_proxmox`, no need for it.

Comment (MasterChief-John-117) [reply proxstar/__init__.py]: IMO that's enough of a deviation from this PR to warrant its own. I can go through all the code at a later date and remove any orphaned calls like this one.

**Pull request merged**